joy_teleop/package.xml

```diff
@@ -16,5 +16,6 @@
   <exec_depend>rospy</exec_depend>
   <exec_depend>rostopic</exec_depend>
   <exec_depend>teleop_tools_msgs</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-future</exec_depend>
```

Comment:
If the whole `future` module is only used for `iteritems()` I would just drop the whole future and use `.items()`. This works for python2&3.
The performance impact will be negligible for the list sizes we are talking about here. Plus there is an initial impact of installing and adding a whole module.

Reply:
That is very good insight, thanks very much. I've adapted the PR